tools/appveyor.psm1

```diff
@@ -29,6 +94,21 @@ function Invoke-AppVeyorBuild
 # Implements the AppVeyor 'install' step
 function Invoke-AppVeyorInstall
 {
+    if(Test-DailyBuild){
+        $buildName = "[Daily]"
+        if($env:APPVEYOR_PULL_REQUEST_TITLE)
+        {
+            $buildName += $env:APPVEYOR_PULL_REQUEST_TITLE
+        }
+        else
+        {
+            $buildName += $env:APPVEYOR_REPO_COMMIT_MESSAGE
+        }
+
+        Update-AppveyorBuild -message $buildName
+    }
+
+    Set-BuildVariable -Name TestPassed -Value True
```

Comment:
could you please help me understand why we are setting `$env:TestPassed` to be `true` at the installation phase?